packages/fxa-auth-server/lib/senders/emails/fluent-localizer.ts

```diff
@@ -117,4 +115,4 @@ class FluentLocalizer {
     };
   }
 }
-module.exports = FluentLocalizer;
+export default FluentLocalizer;
```

Comment:
eslint check did not pass because of this change, so I tweaked the parser options in the config file. Refer [this](https://github.com/AtomLinter/linter-eslint/issues/462)